callback.go

```diff
@@ -353,13 +353,27 @@ func callbackRetNil(ctx *C.sqlite3_context, v reflect.Value) error {
 	return nil
 }
 
+func callbackRetAny(ctx *C.sqlite3_context, v reflect.Value) error {
+        cb, err := callbackRet(v.Elem().Type())
```

Comment:
It is possible that the callback returned `nil`, in which case `v.Elem()` will return the invalid value. Presumably that needs to translate to SQLite `NULL`.

Reply:
Do you mean I should have a check after the err check like

```
if cb == nil {
  return nil
}
```

?

Reply:
I think it would be this:
```go
if v.IsNil() {
    C.sqlite3_result_null(ctx)
    return nil
}

ve := v.Elem()
cb, err := callbackRet(ve.Type())
if err != nil {
    return err
}
return cb(ctx, ve)
```

Reply:
Applied, thank you!